Makefile.am.service.inc

```diff
@@ -41,7 +41,6 @@ app_PYTHON += \
 	eoscompanion/responses.py \
 	eoscompanion/routes.py \
 	eoscompanion/service.py \
-	eoscompanion/util.py \
```

Comment:
I don't understand why this change is necessary?

Reply:
As discussed in the standup, as I remove the util.py, it only contained the custom logger.